crates/web-sys/src/features/gen_BlobPropertyBag.rs

```diff
@@ -11,10 +11,27 @@ extern "C" {
     #[doc = "*This API requires the following crate features to be activated: `BlobPropertyBag`*"]
     pub type BlobPropertyBag;
     #[cfg(feature = "EndingTypes")]
+    #[doc = "Get the `endings` field of this object."]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `BlobPropertyBag`, `EndingTypes`*"]
+    #[wasm_bindgen(method, getter = "endings")]
+    pub fn get_endings(this: &BlobPropertyBag) -> Option<EndingTypes>;
+    #[cfg(feature = "EndingTypes")]
+    #[doc = "Change the `endings` field of this object."]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `BlobPropertyBag`, `EndingTypes`*"]
     #[wasm_bindgen(method, setter = "endings")]
-    fn endings_shim(this: &BlobPropertyBag, val: EndingTypes);
+    pub fn set_endings(this: &BlobPropertyBag, val: EndingTypes);
+    #[doc = "Get the `type` field of this object."]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `BlobPropertyBag`*"]
+    #[wasm_bindgen(method, getter = "type")]
+    pub fn get_type_(this: &BlobPropertyBag) -> Option<String>;
+    #[doc = "Change the `type` field of this object."]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `BlobPropertyBag`*"]
     #[wasm_bindgen(method, setter = "type")]
-    fn type__shim(this: &BlobPropertyBag, val: &str);
+    pub fn set_type_(this: &BlobPropertyBag, val: &str);
```

Comment:
This should just be named `set_type`, not `set_type_`. The code that does it is here: https://github.com/rustwasm/wasm-bindgen/blob/9aa86a6ecbcc5d6f47e335f7bcae3e2b62ff852a/crates/webidl/src/lib.rs#L437

The `rust_ident` call needs to be moved to after adding on the `set_` (and `get_`).

Reply:
Done.